### ISSUE TITLE
Feature request: Open dialogue that allows filtering based upon draft/schedule published

### ISSUE THREAD

**Issue by mjgraves** [OpenLiveWriter/OpenLiveWriter#420]
At present the app doesn't differentiate between published, scheduled and draft blog posts when using the file open dialogue. It would be great if it could be smarter about this. It's often difficult to find a old draft because of all the other things that appear in the list of posts.

**Comment by Laronski** [comment]
Assuming this is on WordPress initially. I like this idea.

Open Live Writer has a recent posts and drafts option, but it's only for those published or drafted with Open Live Writer. Extending this to when the blog is opened/selected would be helpful.

**Comment by mjgraves** [comment]
Yes, I use Scott Hanselman's recommended trick of syncing the drafts between computer via DropBox.

http://www.hanselman.com/blog/SyncingWindowsLiveWriterDraftsToTheCloudDropboxAndOtherBugFixes.aspx

But that doesn't help when trying to open a post that's been pushed to WP. Some forms of media handling just aren't possible in OLW, so a post may evolve over time, including editing online and offline.

**Comment by jdhitsolutions** [comment]
I would certainly like an option to open an online draft from a Wordpress blog into OLW. I often have a draft started online that I'd like to pick up in the app.

**Comment by ScottIsAFool** [comment]
@jdhitsolutions This can be done already. I created Test draft post in the wordpress dashboard, then open OLW and I can open it straight up
![image](https://cloud.githubusercontent.com/assets/1658896/14831504/fedaddf8-0bed-11e6-878d-5d0da4ddcf7c.png)

**Comment by jdhitsolutions** [comment]
I've never seen that dialog before but took a chance and tried Ctrl+O and sure enough there it is. Don't know why I never saw this before but this will work. Thanks.

**Comment by ScottIsAFool** [comment]
File > open recent/local post would also take you to that, you then just needed to select your blog.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
